src/es_archiver/lib/indices/create_index_stream.js

```diff
@@ -46,14 +46,22 @@ export function createCreateIndexStream({ client, stats, skipExisting, log }) {
 
     async function attemptToCreate(attemptNumber = 1) {
       try {
+        const needsDefaultSpace = index.startsWith('.kibana') && await hasDefaultSpace({ index, client });
```

Comment:
Shouldn't this be `await !hasDefaultSpace({ index, client });`?

Reply:
We are checking if the space exists before the deleting the index, so we know if we should recreate it after we delete/create it. So I don't think so.

Reply:
Turns out that checking for the existence of the default space isn't enough because the .kibana index might be deleted via `esArchiver.unload('some_kibana')`, so I've updated the esArchiver to take a `kibanaUrl` param and check for the spaces plugin in Kibana via `/api/status`.

Reply:
Gotcha, that makes sense in that context.